packages/webapp/pages/squads/[handle]/index.tsx

```diff
@@ -211,7 +211,7 @@ const SquadPage = ({
 };
 
 SquadPage.getLayout = getLayout;
-SquadPage.layoutProps = mainFeedLayoutProps;
+SquadPage.layoutProps = { ...mainFeedLayoutProps, canGoBack: true };
```

Comment:
Did only the squad page and dev card need this addition?

Reply:
Yep. Based from their feedback.